sdk/src/main/java/com/qonversion/android/sdk/Qonversion.kt

```diff
@@ -64,6 +64,11 @@ interface Qonversion {
         }
     }
 
+    /**
+    Call this function to sync the subscriber data with the first launch when Qonversion is implemented.
```

Comment:
I thing would be good to mention here that clients shouldn't think about handling the first time - that logic is encapsulated.